vxquery-xtest/src/main/java/org/apache/vxquery/xtest/XTestOptions.java

```diff
@@ -52,4 +52,7 @@
 
     @Option(name = "-htmlreport", required = false, usage = "HTML Report output file")
     String htmlReport;
-}
\ No newline at end of file
+    
+    @Option(name = "-hdfs", required = false, usage = "run HDFS tests")
+    boolean hdfs;
```

Comment:
Yeah I am still working on the temp dfs cluster for tests, do you want to add another test for single document read from hdfs or just the collection test is fine?